doc/doxygen/src/creating-an-application.md

```diff
@@ -0,0 +1,93 @@
+Creating an application                              {#creating-an-application}
+=======================
+
+[TOC]
+
+To create your own application you need to create a directory containing one or
+multiple C file(s) with your source code and a Makefile. A template Makefile is
+available in the `dist` folder of the
+[RIOT repository](https://github.com/RIOT-OS/RIOT).
+
+The main function                                          {#the-main-function}
+=================
+After the board is initialized, RIOT starts two threads: the idle thread and 
+the main thread. The idle thread has the lowest priority and will run whenever 
+no other thread is ready to run. It will automatically use the lowest possible 
```

Comment:
"Documentation is also specification, not just fact stating" I counter to that ^^